admin-ui/src/main/java/io/crate/rest/action/admin/AdminUIFrontpageAction.java

```diff
@@ -24,29 +24,70 @@
 
 package io.crate.rest.action.admin;
 
+import com.google.common.collect.ImmutableSet;
+import io.crate.rest.CrateRestMainAction;
 import org.elasticsearch.client.Client;
 import org.elasticsearch.common.inject.Inject;
 import org.elasticsearch.common.settings.Settings;
 import org.elasticsearch.rest.*;
 
+import java.util.Set;
+
 import static org.elasticsearch.rest.RestRequest.Method.GET;
 
 /**
  * RestHandlerAction to return a html page containing informations about crate
  */
 public class AdminUIFrontpageAction extends BaseRestHandler {
+    private final CrateRestMainAction crateRestMainAction;
+    private final RestController controller;
+    public static final Set<String> USER_AGENTS = ImmutableSet.of(
+        "Mozilla/6.0",
+        "Mozilla/5.0",
+        "Mozilla/4.0",
+        "Mozilla/3.0",
+        "Opera"
```

Comment:
this is the list of all the browser user agents http://www.useragentstring.com/pages/useragentstring.php?typ=Browser Most of them use Mozilla engine or Opera for Opera browser.. i'm not sure if it makes sense to include the other browsers like http://www.dillo.org/, links, lynx etc. I don't think the admin-ui will work with these browser because angular doesn't officially support them https://docs.angularjs.org/misc/faq